### PR TITLE
Snake DONE

### PR DESCRIPTION
**Pull request by JeniaGaidukov**
- [DEMO LINK](https://JeniaGaidukov.github.io/layout_snake/)
- [TEST REPORT LINK](https://JeniaGaidukov.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
+      content="IE=edge">
+    <meta name="viewport"
+      content="width=device-width, initial-scale=1.0">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="box box_1">1</div>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:16]:
```suggestion
      <div class="box box--1">1</div>
```

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Screenshot 2022-04-04 at 11 47 36
Good job, but is it the correct order? Fix media queries
And you can use just two media there: before 600px and before 900px. Try to rewrite

File: src/index.html (hunk 1 above)
Comment (JeniaGaidukov) [reply src/index.html]: Перевірте будь-ласка, виправив, дякую!

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+      <div class="box box--1">1</div>
+      <div class="box box--2">2</div>
+      <div class="box box--3">3</div>
+      <div class="box box--4">4</div>
+      <div class="box box--5">5</div>
+      <div class="box box--6">6</div>
+    </div>
+  </body>
+
 </html>
```
Comment (marina-tilniak) [inline src/index.html:25]:
```suggestion
  </body>
</html>
```

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+
+  <head>
```
Comment (marina-tilniak) [inline src/index.html:4]: same here

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**